Automattic/simplenote-ios
# Disabling custom transitions: Part 2

### Details:
This is Part II of PR #597. In this PR we're just removing code that's, as of now, unused.

Ref. #507

cc @bummytime 
Thank you!!

### Test
1. Launch Simplenote
2. Press over any note 

- [x] Verify the Editor is presented with a Push transition

### Release
These changes do not require release notes.
